packages/visual-dev/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react'
+import styled, { CSSProp } from 'styled-components'
+import { IconKey, Icon } from '../Icon'
+import * as Styles from './Styles'
+
+type DropdownProps = OptionProps & StyleProps & React.ComponentProps<'div'>
+
+interface OptionProps {
+  text?: string
+  showMenu?: boolean
+  pill?: boolean
+  center?: boolean
+  narrow?: boolean
+  disabled?: boolean
+  icon?: IconKey
+  onClickDropdown?: () => void
+  children?: React.ReactNode
+}
+
+interface ButtonProps {
+  pill?: boolean
+  center?: boolean
+  narrow?: boolean
+  disabled?: boolean
+}
+
+interface DropdownItemProps {
+  onClick?: () => void
+  css?: CSSProp
+  children?: React.ReactNode
+}
+
+interface DropdownSublistProps {
+  name: string
+  css?: CSSProp
+  children: React.ReactNode
+}
+
+interface StyleProps {
+  css?: CSSProp
+}
+
+const DropdownContainer = styled('div')<Required<StyleProps>>`
+  ${Styles.base}
+  ${(props) => props.css}
+`
+
+const DropdownButton = styled('div')<Required<ButtonProps>>`
+  ${Styles.button}
+  border-radius: ${(props) => (props.pill ? '100px' : '4px')};
+  text-align: ${(props) => (props.center ? 'center' : 'left')};
+  line-height: ${(props) => (props.narrow ? '10px' : '16px')};
+  color: ${(props) => (props.disabled ? '#A6B9BD' : '#575757')};
+  background: ${(props) => (props.disabled ? '#F6F6F6' : '#FFFFFF')};

```

Comment:
Colours should use CSS variables

Reply:
They now do (also in the styles.ts file too)
```
text-align: ${(props) => (props.center ? 'center' : 'left')};
line-height: ${(props) => (props.narrow ? '10px' : '16px')};
color: ${(props) => (props.disabled ? 'var(--sq-action-secondary-border)' : 'var(--sq-text-on-secondary)')};
background: ${(props) => (props.disabled ? 'var(--sq-surface-subdued)' : 'var(--sq-surface)')};
&:hover {
  ${(props) => !props.disabled && 'box-shadow: inset 0 0 0 1px var(--sq-action-secondary-hovered);'}
```